### PR TITLE
feat(banner): add banner type in impression item.

### PR DESCRIPTION
**Pull request by LiuDongxu-Vungle**
# Context
Check open-rtb spec 2.5 Section 3.2.6
https://www.iab.com/wp-content/uploads/2016/03/OpenRTB-API-Specification-Version-2-5-FINAL.pdf
Banner types annotates the parent impression as an banner impression. This PR will add it in the library.

# Changes

- Add Banner type to Impression elements. 

- Update UT accordingly.

### CODE REVIEW

File: openrtb/banner.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+// See OpenRTB 2.3.1 Sec 3.2.3.
+//go:generate easyjson $GOFILE
+//easyjson:json
+type Banner struct {
+	Width             int         `json:"w,omitempty"`
+	Height            int         `json:"h,omitempty"`
+	ID                *string      `json:"id,omitempty"`
+	BlockedTypes      []int       `json:"btype,omitempty"`
+	BlockedAttributes []int       `json:"battr,omitempty"`
+	Position          *int         `json:"pos,omitempty"`
```
Comment (yingchen0706v) [inline openrtb/banner.go]: Should we use AdPosition as type here?

File: openrtb/banner_test.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+		}
+
+		if !reflect.DeepEqual(testCase.banner, bannerCopy) {
+			banner1JSON, _ := json.MarshalIndent(testCase.banner, "", "  ")
+			banner2JSON, _ := json.MarshalIndent(bannerCopy, "", "  ")
+			t.Errorf("Banners should hold the same values.\nExpected: %s\n Got: %s", banner1JSON, banner2JSON)
+		}
+
+		if diff := deep.Equal(testCase.banner, bannerCopy); diff != nil {
+			t.Error(diff)
```
Comment (yingchen0706v) [inline openrtb/banner_test.go]: Missing description here.

Comment (LiuDongxu-Vungle) [reply openrtb/banner_test.go]: The Deep library could output the different.

File: openrtb/banner.go [hunk 3]
```diff
@@ -0,0 +1,69 @@
+package openrtb
+
+// Banner types annotates the parent impression as an banner impression.
+// See OpenRTB 2.3.1 Sec 3.2.3.
+//go:generate easyjson $GOFILE
+//easyjson:json
+type Banner struct {
+	Width             int         `json:"w,omitempty"`
+	Height            int         `json:"h,omitempty"`
```
Comment (xiaopeng0216) [inline openrtb/banner.go]: please add `wmax` `hmax` `wmin` `hmin`

**Comment by xiaopeng0216** [comment]
LGTM :trollface:

**Pull request merged**